veloroutes_voies_vertes/install/sql/veloroutes/20_TABLE_SEQUENCE_DEFAULT.sql

```diff
@@ -0,0 +1,573 @@
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- poi_tourisme_id_local_seq
+ALTER SEQUENCE veloroutes.poi_tourisme_id_local_seq OWNED BY veloroutes.poi_tourisme.id_local;
+
+
+-- poi_tourisme_val
+CREATE TABLE veloroutes.poi_tourisme_val (
+    id integer NOT NULL,
+    code text NOT NULL,
+    libelle text
+);
+
+
+-- poi_tourisme_val_id_seq
+CREATE SEQUENCE veloroutes.poi_tourisme_val_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- poi_tourisme_val_id_seq
+ALTER SEQUENCE veloroutes.poi_tourisme_val_id_seq OWNED BY veloroutes.poi_tourisme_val.id;
+
+
+-- portion_val
+CREATE TABLE veloroutes.portion_val (
+    id integer NOT NULL,
+    code text NOT NULL,
+    libelle text
+);
+
+
+-- portion_val_id_seq
+CREATE SEQUENCE veloroutes.portion_val_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- portion_val_id_seq
+ALTER SEQUENCE veloroutes.portion_val_id_seq OWNED BY veloroutes.portion_val.id;
+
+
+-- repere_val
+CREATE TABLE veloroutes.repere_val (
+    id integer NOT NULL,
+    code text NOT NULL,
+    libelle text
+);
+
+
+-- repere_val_id_seq
+CREATE SEQUENCE veloroutes.repere_val_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- repere_val_id_seq
+ALTER SEQUENCE veloroutes.repere_val_id_seq OWNED BY veloroutes.repere_val.id;
+
+
+-- revetement_val
+CREATE TABLE veloroutes.revetement_val (
+    id integer NOT NULL,
+    code text NOT NULL,
+    libelle text
+);
+
+
+-- revetement_val_id_seq
+CREATE SEQUENCE veloroutes.revetement_val_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- revetement_val_id_seq
+ALTER SEQUENCE veloroutes.revetement_val_id_seq OWNED BY veloroutes.revetement_val.id;
+
+
+-- statut_segment_val
+CREATE TABLE veloroutes.statut_segment_val (
+    id integer NOT NULL,
+    code text NOT NULL,
+    text text
+);
+
+
+-- statut_segment_val_id_seq
+CREATE SEQUENCE veloroutes.statut_segment_val_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- statut_segment_val_id_seq
+ALTER SEQUENCE veloroutes.statut_segment_val_id_seq OWNED BY veloroutes.statut_segment_val.id;
+
+
+-- element id
+ALTER TABLE ONLY veloroutes.element ALTER COLUMN id SET DEFAULT nextval('veloroutes.element_id_seq'::regclass);
+
+
+-- etape id
+ALTER TABLE ONLY veloroutes.etape ALTER COLUMN id SET DEFAULT nextval('veloroutes.etape_id_seq'::regclass);
+
+
+-- etat_avancement_val id
+ALTER TABLE ONLY veloroutes.etat_avancement_val ALTER COLUMN id SET DEFAULT nextval('veloroutes.etat_avancement_val_id_seq'::regclass);
+
+
+-- itineraire id_local
+ALTER TABLE ONLY veloroutes.itineraire ALTER COLUMN id_local SET DEFAULT nextval('veloroutes."ItineraireCyclable_id_seq"'::regclass);
```

Comment:
```suggestion
ALTER TABLE ONLY veloroutes.itineraire ALTER COLUMN id_local SET DEFAULT nextval('veloroutes.itineraire_id_local_seq'::regclass);
```
Même problème de serial